includes/Core/Feature_Tours/Dismissed_Tours.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Feature_Tours\Dismissed_Tours
+ *
+ * @package   Google\Site_Kit\Core\Feature_Tours
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Feature_Tours;
+
+use Closure;
+use Google\Site_Kit\Core\Storage\User_Setting;
+
+/**
+ * Class for representing a user's dismissed feature tours.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Dismissed_Tours extends User_Setting {
+
+	/**
+	 * The user option name for this setting.
+	 *
+	 * @note This option is prefixed differently
+	 * so that it will persist across disconnect/reset.
+	 */
+	const OPTION = 'googlesitekitpersistent_dismissed_tours';
+
+	/**
+	 * Adds one or more tours to the list of dismissed tours.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string ...$tour_slug The tour identifier to dismiss.
+	 */
+	public function add( ...$tour_slug ) {
```

Comment:
This doesn't seem necessary at this point. I'm not opposed to keeping it, but in general I prefer separate explicit methods for one vs multiple.

Reply:
Are you referring to the method as a whole or just the ability to add more than one at a time?

I see this as similar to `Settings::merge` in that you can set multiple "inner values" in one go, rather than requiring a separate DB query for each addition. The method body is basically the same to add a single value as it is for multiple as well.

Reply:
I'm not opposed to having a method to dismiss multiple tours in one go (one DB update), but I meant that I'd prefer to have a separate method to be more explicit in usage. `Module_Settings::merge` only works with an array of values, while this here supports any number of values. I guess that's just a personal preference, but I'm not a large fan of using variable parameters for this purpose, rather would go with `add( $tour_slug )` and something like `add_multiple( $tour_slugs )`; or alternatively not have anything for single and only have `add( $tour_slugs )`.

The way it is now is okay, my other point was also that we just don't need it at this point since the API doesn't support dismissing multiple tours at once anyway, so we could basically avoid this discussion and only add `add( $tour_slug )` for now. But let's go with this as is, not heavily opposed.